Author: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>

Since there is no GTK+2 package for Tizen 2.1 Mobile and we already use Aura
instead of GTK+2 directly, there should be no need to look for GTK+2 at all.

Upstreaming this patch depends on a hard GTK+2 dependency on
remoting/remoting.gyp being removed in chromium. It does not affect crosswalk
because we do not have any dependency on remoting.gyp.

Upstream patch: https://codereview.chromium.org/19531008
Depends on:     https://code.google.com/p/chromium/issues/detail?id=247213
--- src/build/linux/system.gyp
+++ src/build/linux/system.gyp
@@ -29,7 +29,7 @@
         'use_system_ssl%': 1,
       },
     }],
-    [ 'chromeos==0', {
+    [ 'chromeos==0 and toolkit_uses_gtk==1', {
       # Hide GTK and related dependencies for Chrome OS, so they won't get
       # added back to Chrome OS. Don't try to use GTK on Chrome OS.
       'targets': [
